index.js

```diff
@@ -19,14 +19,24 @@ function convertDate (d: Date): number {
     (d.getHours() << 11) | (d.getMinutes() << 5) | (d.getSeconds() >> 1)
 }
 
+// Filter hidden files/directories.
+function filterHidden (item: string) {
+  const basename = path.basename(item)
+  return basename === '.' || basename[0] !== '.'
```

Comment:
I'm not sure how this behavior should be; is `.` a hidden path or not?

Reply:
`.` refers to the current directory which needs to be included in the zip file. It is not hidden.

Reply:
Duh, forgot about the fact that `.` is a special path. You can't actually name something `.`.